ci/setup_thrift.ps1

```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = "Stop"
+trap { $host.SetShouldExit(1) }
+
+# TODO: consider moving this to .gitmodules
+git clone https://github.com/Microsoft/vcpkg.git
```

Comment:
Checkout a stable release tag for reproducible build?

Reply:
done.